src/lib/gitutils.ts

```diff
@@ -301,3 +301,17 @@ export const checkoutCommitPushCreatePRLink = async (
     throw err;
   }
 };
+
+/**
+ * Validates whether a url is a github url
+ * TEMPORARY, UNTIL GITHUB REPOS ARE SUPPORTED
+ *
+ * @param url the url string
+ */
+export const isGitHubUrl = (url: string) => {
+  const gitUrl = url.includes("github");
```

Comment:
I think we should use a url parser here instead of string matching ie https://nodejs.org/api/url.html#url_url_hostname

Reply:
@bnookala , we have followed your suggestion on parsing the url string and check for host name.